### DISCUSSION TITLE
Dithering artefact with 2x and 3x upscale modes

### DISCUSSION THREAD

**Discussion by intracube-paul** [upscayl/upscayl#481] [Q&A] [answered]
Hi, when using either the 2x or 3x modes, the upscaled image has noticeable dithering compared to 4x mode. Has anyone noticed this or knows what the cause is?

It looks a bit like 4 bit dithered. Although a histogram shows that the colours aren't actually grouped in 16 levels.

It seems to happen on all models.

Edit: This is with Upscayl 2.8.1 on Linux.
Edit 2: I'm actually not sure what is causing this, as I now see it on some 4x upscale content. Perhaps it's not visible when outputting JPG images as the compression hides it, but with PNG it's visible.

Attached are the original 400x300 sample, the 2x upscale and crop of 2x upscale:
![sample_400x300](https://github.com/upscayl/upscayl/assets/12416914/9e989585-1267-4477-8ad1-43d825159145)
![sample_400x300_upscayl_2x_realesrgan-x4plus](https://github.com/upscayl/upscayl/assets/12416914/e01ce32c-3c1a-4488-b69a-eaac75588317)
![sample_400x300_upscayl_2x_realesrgan-x4plus_crop_zoom](https://github.com/upscayl/upscayl/assets/12416914/9daaee5b-bea3-4b06-9040-917ac45d65c4)

**Comment by aaronliu0130** [comment, marked as answer]
Are you sure this isn’t #474, where every Upscayl type has similar stuff?
If you’re sure, would you kindly submit a bug report?

**Comment by intracube-paul** [comment]
> Are you sure this isn’t #474, where every Upscayl type has similar stuff? If you’re sure, would you kindly submit a bug report?

Ah, sorry. I searched for dithering and had a quick look at the bug list, but missed that.

I haven't tested a build of the latest code but this definitely looks to be the same issue, so I'll mark this as resolved and check with the next release.
